### PR TITLE
Codespaces setup

### PR DESCRIPTION
**Pull request by JamesMGreene**
Setting up GitHub Codespaces for the backend repo!

### CODE REVIEW

**Comment by andyfeller** [comment]
@andyfeller

File: knexfile.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 const NODE_ENV = process.env.NODE_ENV || 'development'
 const isProduction = NODE_ENV === 'production'
+const POSTGRES_PORT = parseInt(process.env.POSTGRES_PORT, 10) || undefined
```
Comment (andyfeller) [inline knexfile.js:3]: @JamesMGreene : shouldn't this be `5443`?

**Comment by manishapriya94** [comment]
From @brunoguardia 
[Managing encrypted secrets for your codespaces - GitHub Docs](https://docs.github.com/en/codespaces/managing-your-codespaces/managing-encrypted-secrets-for-your-codespaces)
You can store sensitive information, like tokens, that you want to access in your codespaces via environment variables.
You can define a few variables and describe in the README.md the setup for each person to configure them in their Codespaces; and make the script validate so if they are empty the script invites the user to follow the process in the README

File: knexfile.js (hunk 1 above)
Comment (JamesMGreene) [reply knexfile.js:5]: @andyfeller If you're referring to the `10`, that's a confusing antiquated pattern in the [JS `parseInt` method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt) where the second parameter (radix) is the intended base (i.e. `10` = "parse this as a base 10 decimal number"). It's not strictly required anymore but some linters still get angry if you leave it off. 🙄 

If you're referring to the `undefined`, that is intentional to ensure that local development outside of Codespaces can still use the default PostgreSQL port (`5442`) for a local database instance.

**Comment by JamesMGreene** [comment]
I have configured this repository with the following Codespaces Secrets:

```
# Auth0
AUTH0_AUDIENCE=your_Auth0_identifier_value
AUTH0_DOMAIN=your_Auth0_domain
CLIENT_ORIGIN_URL=http://localhost:4040
SERVER_PORT=6060

# Google Civic API
CIVIC_API_KEY=(James's restricted test API key)
TEST_CIVIC_API_KEY=(James's restricted test API key)

# Lob API
LOB_API_KEY=(our Lob test API key)
TEST_LOB_API_KEY=(our Lob test API key)

# Stripe API
STRIPE_SECRET_KEY=(our Stripe test API key)
TEST_STRIPE_SECRET_KEY=(our Stripe test API key)

# General
NODE_ENV=development
```

**Comment by manishapriya94** [comment]
@andyfeller @moniarchy if you have the bandwidth to review? 🌻

**Comment by JamesMGreene** [comment]
I haven't changed much since the previous review other than adding a bit more documentation. I'm going to move ahead and merge this so I can test out the Codespaces before tomorrow's hackathon but I would still welcome reviews and/or improvements afterward. 🙇🏻‍♂️ 🙏🏻

**Pull request merged**